src/inc/msquic.h

```diff
@@ -362,6 +362,7 @@ typedef enum QUIC_CIPHER_SUITE {
 
 typedef enum QUIC_CONGESTION_CONTROL_ALGORITHM {
     QUIC_CONGESTION_CONTROL_ALGORITHM_CUBIC,
+    QUIC_CONGESTION_CONTROL_ALGORITHM_BBR,
```

Comment:
Wrap with `#ifdef QUIC_API_ENABLE_PREVIEW_FEATURES`
```suggestion
#ifdef QUIC_API_ENABLE_PREVIEW_FEATURES
    QUIC_CONGESTION_CONTROL_ALGORITHM_BBR,
#endif
```